File: JAVA/DSA in java/Searching in java/linearsearch.java
public class linearsearch {
    public static int linea(int numbers[],int key){
        for(int i =0;i<numbers.length;i++){
            if(numbers[i]==key){
                return i;
            }
        }
        return -1;
    }
    public static void main(String args[]){
        int numbers[]={2,4,6,8,10,12,14,16};
        int key=10;
        int index=linea(numbers,key);
        if(index==-1){
            System.out.print("not");
        }
        else{
            System.out.print("key is at index " + index);
        }
    }
}

File: JAVASCRIPT/Algorithms/linearSearch.js
const linearSearch = (arr, searchElement) => {
  for (let index = 0; index < arr.length; index++) {
    if (arr[index] === searchElement) return index;
  }
  return -1;
};

// console.log(linearSearch([28, 59, 68, 16], 5)); // Output: -1
// console.log(linearSearch([28, 59, 68, 16], 68)); // Output: 2
